DataManagementSystem/Client/FTS3Operation.py

```diff
@@ -384,7 +383,13 @@ def prepareNewJobs( self, maxFilesPerJob = 100, maxAttemptsPerFile = 10 ):
 
     for targetSE, ftsFiles in filesGroupedByTarget.iteritems():
 
-      res = self._checkSEAccess( targetSE, 'WriteAccess' )
+      voName = None
+      if ftsFiles:
+        res = _getVoName(ftsFiles[0].lfn)
+        if res['OK']:
+          voName = res['Value']
+
+      res = self._checkSEAccess(targetSE, 'WriteAccess', voName=voName)
```

Comment:
These `voName =` won't work. Anyway RSS is not VOs aware...

Reply:
For completeness: https://github.com/DIRACGrid/DIRAC/issues/3260

Reply:
Also I am not quite sure what it brings ?

Reply:
If you look into the _checkSEAccess function, the code to get SE Status from RSS was commented out by @chaen . StorageElement's getStatus function is used instead.

Reply:
Should we get the SE status from RSS?

Reply:
getStatus does the job for you so this is fine. Now indeed, the SE should know the VO in case of multiVO, however RSS will not make any difference.

Reply:
Can you please explain me again why do you need to get the VO name from here and what does it bring ? I am still not sure I undestand

Reply:
Still this is to be fixed.

Reply:
i have removed the voName arg, there is no need for it...The intuition behind using it was that, as i am using voName in some other parts of the code, it will improve the StorageElement Cache hit, otherwise new SE obj will be created without the voName..